data/huion-h420.tablet

```diff
@@ -0,0 +1,23 @@
+# HUION
+# H420
+#
+
+[Device]
+Name=Huion H420
+DeviceMatch=usb:256c:006e:HUION H420 Pen;usb:256c:006e:HUION H420 Pad;usb:256c:006e:HUION H420 Consumer Control;usb:256c:006e:HUION H420 System Control;usb:256c:006e:HUION H420 Keyboard;usb:256c:006e:HUION H420 Mouse;
```

Comment:
Pretty sure only one of those is your actual pen device, the others may appear but they're not tablets. The "Pen" and "Pad" ones should be enough here.

Reply:
Potential feature: adding the ability to use globs here...

Reply:
Also DeviceMatch should be ok now